db/schema.rb

```diff
@@ -120,12 +120,6 @@
     t.index ["name"], name: "index_ext_apps_on_name", unique: true
   end
 
-  create_table "group_roles", force: :cascade do |t|
```

Comment:
this is not used anymore?

Reply:
This was auto generated when we ran `rails db:migrate`. Maybe `schema.rb` was outdated by couple of commits.

Reply:
can someone confirm this is expected or not?

Reply:
After checked and review previous version. this should be correct mas. we still use `instances` for cluster_templates and this is part will be deleted on `cleanup legacy code` task.
and the `group_roles` there is shouldn't be here since we use `app_group_roles`, after run it on local, this also be deleted.